nginx/modsecurity/apr_bucket_nginx.c

```diff
@@ -168,12 +106,10 @@ move_chain_to_brigade(ngx_chain_t *chain, apr_bucket_brigade *bb, ngx_pool_t *po
         if (chain->buf->last_buf) {
             e = apr_bucket_eos_create(bb->bucket_alloc);
             APR_BRIGADE_INSERT_TAIL(bb, e);
-            chain->buf->last_buf = 0;
             return NGX_OK;
         }
-        cl = chain;
+
         chain = chain->next;
-        ngx_free_chain(pool, cl);
```

Comment:
because I don't want to touch the nginx structures - only read from them. This way fewer things go wrong. Our use of modsec does not do anything that rewrites the request anyway.